src/sagemaker/model.py

```diff
@@ -79,6 +79,7 @@ def __init__(
         vpc_config=None,
         sagemaker_session=None,
         enable_network_isolation=False,
+        output_kms_key=None,
```

Comment:
docstring? (same for the rest)

Reply:
will add

Reply:
`**kwargs` also need docstring entries - if possible, include a link to where the `**kwargs` are eventually sent to

Reply:
k